templates/product_info.html

```diff
@@ -0,0 +1,61 @@
+{% extends "base.html" %}
+{% from "jinja_macros/product_listing.html" import list_container %}
+{% from "jinja_macros/product_listing.html" import listing %}
+
+
+{% block content %}
+    <div class="columns">
+        <div class="column is-3 is-fixed center">
+            <aside class="menu is-centered">
+                {% block sidebar %}
```

Comment:
This block will not do anything since it's already blocked by the block content call

Reply:
Fixed.